test/Builder.test.ts

```diff
@@ -444,6 +444,67 @@ describe('QueryBuilder', () => {
     )
   })
 
+  test('select with subquery join', async () => {
+    const query = new QuerybuilderTest()._select({
+      tableName: 'testTable',
+      fields: '*',
+      where: {
+        conditions: 'field = ?1',
+        params: ['test'],
+      },
+      join: {
+        table: {
+          tableName: 'otherTable',
+          fields: ['test_table_id', 'GROUP_CONCAT(attribute) AS attributes'],
+          groupBy: 'test_table_id',
+          alias: 'otherTableGrouped',
```

Comment:
and the tests would also need to be updated to move the alias property to the outer object